src/style/app.module.css

```diff
@@ -5963,17 +6785,17 @@ button[data-testid='createPostBtn'] {
 
 .createButtonEventHeader {
   background-color: var(--grey-bg-color);
-  color: var(--brown-color);
+  color: var(--dark-grey-font);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`